packages/flutter/test/material/paginated_data_table_test.dart

```diff
@@ -70,107 +71,118 @@ void main() {
   setUp(() => source = TestDataSource());
   tearDown(() => source.dispose());
 
-  testWidgets('PaginatedDataTable paging', (WidgetTester tester) async {
-    final List<String> log = <String>[];
+  testWidgets(
```

Comment:
See https://github.com/flutter/flutter/pull/146748/files#r1566025452.
Can you also fix indent introduced by https://github.com/flutter/flutter/pull/146747 , here?

Reply:
Sure, let me revert and fix that 👍

Reply:
[Revert "test: Update the tests"](https://github.com/flutter/flutter/pull/146755/commits/4cce57066bfe131d6977d299acb0d6ed6b5e88eb) and [Revert bad formatting in transitions_test.dart](https://github.com/flutter/flutter/pull/146755/commits/0734f55dbdf73c5c880f6bac592e7d8e321b6296) should have taken care of that